pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -6450,6 +6510,81 @@ func (prt PipelineRunTest) reconcileRun(namespace, pipelineRunName string, wantE
 	return reconciledRun, clients
 }
 
+func TestReconcile_RemotePipelineRefValidation_Failure(t *testing.T) {
+	names.TestingSeed()
+
+	namespace := "foo"
+	prName := "test-pipeline-run-failure"
```

Comment:
The test setup here looks like a duplicate from the below `TestReconcile_RemotePipelineRef()` test, is it possible to put it into a helper function?